fairlearn/metrics/_group_metric_set.py

```diff
@@ -98,16 +96,16 @@
 def create_group_metric_set(model_type,
```

Comment:
Do we want to keep calling this `create_group_metric_set`? I can't think of a concise alternative, though. This would also affect what we put in the `schemaType` field.